client/layout/global-sidebar/menu-items/search/icon.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const SearchIcon = () => (
```

Comment:
I think that we should use the search icon from core for this instead: https://wordpress.github.io/gutenberg/?path=/docs/icons-icon--docs